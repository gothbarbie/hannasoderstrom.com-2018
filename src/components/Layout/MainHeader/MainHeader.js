// @flow
import React from 'react'
import { Link } from 'react-router-dom'
import styled from 'styled-components'

import Wrapper from '../Wrapper'
import Logo from '../../Logo'
import MainNavigation from '../MainNavigation'

import imgHeroComments from '../../../images/menu-comments.svg'

const Header = styled.header`
  background: #3a1c71;
`

const HeroComments = styled.img`
  position: absolute;
  bottom: -80px;
  right: 4%;
  width: 350px;
  z-index: 2;
  display: none;

  @media (min-width: 600px) {
    display: block;
  }
`

const MainHeader = () => {
  return (
    <Header>
      <Wrapper>
        <Link to="/">
          <Logo />
        </Link>
        <MainNavigation />
        <HeroComments src={imgHeroComments} />
      </Wrapper>
    </Header>
  )
}

export default MainHeader
