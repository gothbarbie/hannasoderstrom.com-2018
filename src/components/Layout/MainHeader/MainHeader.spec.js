import React from 'react'
import { shallow } from 'enzyme'

import MainHeader from './index'

describe('components/MainHeader', () => {
  let component

  beforeEach(() => {
    component = shallow(<MainHeader />)
  })

  it('renders', () => {
    expect(component).toMatchSnapshot()
  })
})
