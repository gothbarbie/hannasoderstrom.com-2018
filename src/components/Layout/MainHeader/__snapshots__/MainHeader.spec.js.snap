// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/MainHeader renders 1`] = `
<styled.header>
  <styled.div>
    <Logo />
    <MainNavigation
      currentPath="/"
    />
    <styled.img
      src="menu-comments.svg"
    />
  </styled.div>
</styled.header>
`;
